src/app/settings/settings.model.ts

```diff
@@ -2,7 +2,7 @@ import { AppState } from '@app/core';
 
 export const NIGHT_MODE_THEME = 'BLACK-THEME';
 
-export type Language = 'en' | 'sk' | 'de' | 'fr' | 'es' | 'pt-br';
+export type Language = 'en' | 'sk' | 'de' | 'fr' | 'es';
```

Comment:
Could it be that this is a leftover from the rebase, or was it intentionally to remove `pt-br`?